### PR TITLE
Add new nightly case for GC untagged images

### PR DESCRIPTION
**Pull request by AllForNothing**
Result:
image

Signed-off-by: AllForNothing <sshijun@vmware.com>

Fixed issue: #11593

### CODE REVIEW

File: tests/resources/Harbor-Pages/Project-Artifact.robot [hunk 1]
```diff
@@ -41,4 +41,11 @@ Delete A Tag
...
     Retry Wait Until Page Contains Element  ${dialog_delete_button}
-    Retry Element Click  ${dialog_delete_button}
\ No newline at end of file
+    Retry Element Click  ${dialog_delete_button}
+
+Should Contain Artifact
+    [Arguments]  ${tag}
+    Retry Wait Until Page Contains Element   xpath=//artifact-list-tab//clr-dg-row//clr-dg-cell[contains(.,'${tag}')]
+
+Should Contain No Artifact
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Project-Artifact.robot:50]: Should Not Contain Artifact

File: tests/robot-cases/Group1-Nightly/Common.robot [hunk 2]
```diff
@@ -556,6 +556,38 @@ Test Case - Tag CRUD
...
+    ${d}=    Get Current Date    result_format=%m%s
+    Create An New Project  project${d}
+    Push Image With Tag  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  hello-world  latest
+    # make hello-world untagged
+    Go Into Project   project${d}
+    Go Into Repo   hello-world
+    Go Into Artifact   latest
+    Should Contain Tag   latest
+    Delete A Tag   latest
+    # run gc without param delete untagged artifacts checked,  should not delete hello-world:latest
```
Comment (danfengliu) [inline tests/robot-cases/Group1-Nightly/Common.robot:573]: Should Not Contain Tag   latest
Please add this after delete a tag.

Comment (AllForNothing) [reply tests/robot-cases/Group1-Nightly/Common.robot:574]: Fixed

File: tests/robot-cases/Group1-Nightly/Common.robot [hunk 3]
```diff
@@ -557,6 +557,34 @@ Test Case - Tag CRUD
     Delete A Tag   latest
     Should Not Contain Tag   latest
 
+Test Case - GC Untagged Images
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    ${d}=    Get Current Date    result_format=%m%s
+    Create An New Project  project${d}
+    Push Image With Tag  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  hello-world  latest
```
[the remark this answers is not in the document]
Comment (AllForNothing) [reply tests/robot-cases/Group1-Nightly/Common.robot:566]: Fixed

File: tests/resources/Harbor-Pages/Project-Artifact.robot (hunk 1 above)
Comment (AllForNothing) [reply tests/resources/Harbor-Pages/Project-Artifact.robot]: Fixed

**Pull request merged**